src/lib/roles.ts

```diff
@@ -709,8 +804,46 @@ export const roles_definition = validateRoles({
 			"4":"FAN_ONLY",
 			"5":"HEAT",
 			"6":"OFF"
-		}
+		},
+		"unit": "null"
 	},
+	// Vacuum cleaner
+	"level.mode.cleanup": {
+		"desc":"Mode for Vacuum cleaner",
+		"type": [
+			"string",
+			"number"
+		],
+		"read":true,
+		"write":true,
+		"states":{
+			"0":"AUTO",
+			"1":"eco",
+			"2":"express",
+			"3":"normal",
+			"4":"quiet",
+		},
+		"unit": "null"
+	},
+	"level.mode.work": {
+		"desc":"Mode for Vacuum cleaner",
+		"type": [
+			"string",
+			"number"
+		],
+		"read":true,
+		"write":true,
+		"states":{
+			"0":"AUTO",
+			"1":"fast",
+			"2":"medium",
+			"3":"slow",
+			"4":"turbo",
```

Comment:
Is the original enumeration really that chaotic? Maybe we need to change the specs/docs :D

Reply:
Official documentation:
```
level.mode.work - Enumeration of AUTO, FAST, MEDIUM, SLOW, TURBO. Optional state.
```
Documentation is not straight in some point. One more example:

```
Levels (numbers, read-write)
With levels you can control or set some number value.
level.co2 - 0-100% ait quality
```

Didn't know that I can choose the co2-level :)

Reply:
Hmm, IMO that documentation doesn't really imply any particular order. Let's sort it in a sensible way.

> Didn't know that I can choose the co2-level :)

LOL

Reply:
Gona set the order to AUTO, slow, medium, fast, turbo
That makes more sense.